libs/core/kiln_ai/datamodel/extraction.py

```
@@ -78,15 +109,88 @@
 
     @model_validator(mode="after")
     def validate_properties(self) -> Self:
-        if self.extractor_type == ExtractorType.gemini:
+        if self.extractor_type == ExtractorType.GEMINI:
             validate_prompt_for_kind(self.properties.get("prompt_for_kind"))
             validate_model_name(self.properties.get("model_name"))
             return self
         else:
             raise ValueError(f"Invalid extractor type: {self.extractor_type}")
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)